src/lib/hooks/use-current-coaching-relationship.ts

```diff
@@ -3,15 +3,39 @@
 import { useCoachingRelationship } from "@/lib/api/coaching-relationships";
 import { useCoachingRelationshipStateStore } from "@/lib/providers/coaching-relationship-state-store-provider";
 import { useCurrentOrganization } from "./use-current-organization";
+import type { CoachingRelationshipWithUserNames } from "@/types/coaching-relationship";
+import type { EntityApiError } from "@/types/general";
+
+/**
+ * Return type for the useCurrentCoachingRelationship hook.
+ */
+export interface UseCurrentCoachingRelationshipReturn {
```

Comment:
Thank you for formalizing the type here!!

I wonder if `Result` would be more consistent with the rest of the application, than `Return` in the name here?

Reply:
I like it, you're right.

Fixed in commit 27d6e92